cesiumOmniverseCppTestsExtension/include/UsdUtilTests.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+void run_all_UsdUtil_tests();
```

Comment:
Use camelCase for C++ code: `runAllUsdUtilTests`

Reply:
I have appear to have over-corrected for my camelCase tendencies